lms/djangoapps/certificates/queue.py

```diff
@@ -214,6 +214,18 @@ def add_cert(self, student, course_id, course=None, forced_grade=None, template_
         Returns the newly created certificate instance
         """
 
+        if not hasattr(course_id, 'ccx'):
```

Comment:
I am reading this `if` statement and I am really confused. This `if` statement tells me that if the course do not have ccx attribute, which to me means if the course is not of ccx type, then run the statement below which prevents certificate generation. This means to me that we want to prevent all courses, except CCX courses, to generate certificates. Where am I wrong?